examples/go/go.mod

```diff
@@ -1,11 +1,9 @@
-module github.com/twilio/twilio-go
+module github.com/twilio/twilio-oai-generator
```

Comment:
I know that if we want to do a `go get module`, the name should follow the pattern `<host>/repo/path/within/repo`. In this case, we shouldn't be allowing anyone to `go get twilio-oai-generator`. Does it make sense to have this module named so?

Reply:
So renaming this to github.com/twilio/twilio-oai-generator/examples/go?